docs/how-to-guides/meson-args.rst

```diff
@@ -44,6 +44,21 @@ building Python wheel. User options specified via ``pyproject.toml``
 or via Python build front-end config settings can be used to override
 the ``meson-python`` defaults..
 
+When building on Windows, ``meson-python`` invokes the ``ninja``
+command via the ``meson compile`` wrapper. This activates the Visual
```

Comment:
Can you delete the line ``This activates the Visual Studio environment ...``?

This contradicts the sentence right after, which should be the more accurate one (since activation of VS env is not guaranteed).

Reply:
Sure. It should be better now. PTAL